src/TestStack.White/InputDevices/Mouse.cs

```diff
@@ -1,145 +1,24 @@
 using System;
-using System.Collections.Generic;
-using System.Runtime.InteropServices;
-using System.Threading;
 using System.Windows;
-using TestStack.White.Configuration;
-using TestStack.White.Drawing;
-using TestStack.White.UIA;
-using TestStack.White.UIItems;
 using TestStack.White.UIItems.Actions;
-using TestStack.White.WindowsAPI;
 using Action = TestStack.White.UIItems.Actions.Action;
 
 namespace TestStack.White.InputDevices
 {
-    public class Mouse : IMouse
+    public class Mouse : BaseMouse, IMouse
     {
-        [DllImport("user32", EntryPoint = "SendInput")]
-        static extern int SendInput(uint numberOfInputs, ref Input input, int structSize);
-
-        [DllImport("user32", EntryPoint = "SendInput")]
-        static extern int SendInput64(int numberOfInputs, ref Input64 input, int structSize);
-
-        [DllImport("user32.dll")]
-        static extern IntPtr GetMessageExtraInfo();
-
-        [DllImport("user32.dll")]
-
-        static extern bool GetCursorPos(ref System.Drawing.Point cursorInfo);
-
-        [DllImport("user32.dll")]
-        static extern bool SetCursorPos(int x, int y);
-
-        [DllImport("user32.dll")]
-        static extern bool GetCursorInfo(ref CursorInfo cursorInfo);
-
-        [DllImport("user32.dll")]
-        static extern short GetDoubleClickTime();
-
-        [DllImport("user32.dll")]
-        static extern int GetSystemMetrics(SystemMetric smIndex);
-
-        public static Mouse Instance = new Mouse();
```

Comment:
On a side not, I see Fields as Class-Level.
The way the static is being used is more on the side of a Singleton.
That raises the question. Do you want a Singleton or do you want Mouse to be also instanciated?
Serve one or the other.

I would be ok as to making Mouse and Keyboard a singleton pure and through.